Makefile

```diff
@@ -1,17 +1,25 @@
 all:
 	$(MAKE) -C c-layer $@
 	$(MAKE) -C c-example $@
+# TODO: put bindgen to build.rs
+	(cd rust/zypp-agama-sys; bindgen --merge-extern-blocks headers.h -o src/bindings.rs -- -I../../c-layer/include)
 	(cd rust; cargo build)
 	doxygen
 
 check:
+	git ls-files | grep '\.[ch]' | \
+	  xargs --verbose clang-format --style="{BasedOnStyle: llvm, ColumnLimit: 120}" --dry-run
```

Comment:
do we need that wider collumns?

Reply:
I based that on your original code... meanwhile we could even reformat it into 80 columns

Reply:
Ideally I would use default of clang whatever it is as I do not see benefit of differing from upstream.